mrtarget/modules/ChEMBL.py

```diff
@@ -3,41 +3,19 @@
 
 import requests
 import itertools
-import more_itertools
+import shelve
+import dumbdbm
+import tempfile
 
-from mrtarget.common.ElasticsearchQuery import ESQuery
-from mrtarget.common.connection import new_es_client
-from mrtarget.Settings import Config
 from mrtarget.common import URLZSource
 import json
```

Comment:
Better to do `import simplejson as json`. Its already a dependency elsewhere, and should give a performance boost.

Reply:
It makes sense. Thanks

Reply:
About shelve, Adam and I verified that computationally is better use shelve. Adding comment about why I added dumbdbm.